backend/src/main/kotlin/app/ehrenamtskarte/backend/cards/webservice/schema/CardMutationService.kt

```diff
@@ -115,6 +124,88 @@ class CardMutationService {
         }
     }
 
+    @GraphQLDescription("Creates a new digital entitlementcard from self-service portal")
+    fun createCardFromSelfService(
+        dfe: DataFetchingEnvironment,
+        project: String,
+        encodedCardInfo: String,
+        generateStaticCode: Boolean
+    ): CardCreationResultModel {
+        // TODO disable for production until the Koblenz project is ready to go live
+        val context = dfe.getContext<GraphQLContext>()
+        if (context.backendConfiguration.production) {
+            throw ForbiddenException()
+        }
+        if (project != KOBLENZ_PASS_PROJECT) {
+            throw NotKoblenzProjectException()
+        }
+
+        val cardInfo = parseEncodedCardInfo(encodedCardInfo)
+        val regionId = cardInfo.extensions.extensionRegion.regionId
+        transaction { RegionsRepository.findByIdInProject(project, regionId) } ?: throw RegionNotFoundException()
```

Comment:
Maybe it's a redundant check, since we check entitlements per region anyway

Reply:
According to my comment before we shouldn't need it, since we get it from userEntitlements and if the user was not found you already throw an exception